{"uuid":"fd9cc2f0-4ca3-44ce-8157-2b384c63ceb2","name":"Scenario Outline: The user getting error message with invalid python code for Traversal page","children":["77e01385-215e-426c-99be-5d4dd5cd73eb"],"befores":[{"name":"AppHooks.ApplicationHooks.scenario(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706234104376,"stop":1706234104376}],"afters":[{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706234104556,"stop":1706234104556},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706234104705,"stop":1706234104705},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706234105105,"stop":1706234105106},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706234105106,"stop":1706234105106},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706234105404,"stop":1706234105405},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706234105592,"stop":1706234105592},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706234105621,"stop":1706234105621},{"name":"AppHooks.ApplicationHooks.afterstep(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1706234105627,"stop":1706234105627}],"start":1706234104376,"stop":1706234105627}